backend/submodule/peripherals.py

```diff
@@ -1928,5 +1928,5 @@ def compute(self) -> bool:
         print(f'[DEBUG] UART: {io_vcc_aux_power = }', file=sys.stderr)
         print(f'[DEBUG] UART: {self.output.calculated_bandwidth = }', file=sys.stderr)
         print(f'[DEBUG] UART: {self.output.block_power = }', file=sys.stderr)
-
         return True
+    
```

Comment:
fix this

Reply:
the excess space issue right? yes, it will be fixed.